feat: Update vep version

fevac opened the pull request:
## Description

Closes https://github.com/Clinical-Genomics/BALSAMIC/issues/1556

#### Changed

- Update to latest VEP version
- Updated rank model to use gnomad_popmax since gnomAD_AF is no longer in use in the latest VEP version

#### Fixed
- Fixed varcall_py27 container


## Documentation

<!-- Link all added or updated documents. -->

- [ ] N/A
- [ ] Updated Balsamic documentation to reflect the changes as needed for this PR.
    - [Document Name]

## Tests

<!-- Describe in detail how you tested your changes to help reviewers validate the code. -->
<!-- Include screenshots or visual representations of your changes. -->

### Feature Tests

<!-- Include tests relevant to the changes in this PR. -->

- [ ] N/A
- [ ] Test [Description]
    - [Screenshot]

### Pipeline Integrity Tests

<!-- Include tests to verify the integrity of the different Balsamic workflows. -->

- **Report deliver (generation of the `.hk` file)**
    - [x] N/A
    - [ ] Verified
- **TGA T/O Workflow**
    - [x] N/A
    - [ ] Verified
- **TGA T/N Workflow**
    - [x] N/A
    - [ ] Verified
- **UMI T/O Workflow**
    - [x] N/A
    - [ ] Verified
- **UMI T/N Workflow**
    - [x] N/A
    - [ ] Verified
- **WGS T/O Workflow**
    - [x] N/A
    - [ ] Verified
- **WGS T/N Workflow**
    - [x] N/A
    - [ ] Verified
- **QC Workflow**
    - [x] N/A
    - [ ] Verified
- **PON Workflow**
    - [x] N/A
    - [ ] Verified

## Clinical Genomics Stockholm

<!-- Do not reveal clinical data, and if applicable, place it within the internal Google Drive directory. -->

### Documentation

<!-- Link related issues or PRs for necessary changes. -->

- **Atlas documentation**
    - [x] N/A
    - [ ] Updated: [Link]
- **Web portal for Clinical Genomics**
    - [x] N/A
    - [ ] Updated: [Link]

### Panel of Normal specific criteria

<!-- If the PR includes a new PoN adhere to the criteria below. -->

- [ ] The PR includes the addition of a new Panel of Normals
- [ ] The samples have been verified to adhere to the sample selection criteria on [Atlas PoN creation instructions for Balsamic](https://atlas.scilifelab.se/infrastructure/BALSAMIC/balsamic_pon/#procedure)

### User Changes

<!-- Please provide justification if you select N/A and the changes affect the output or results. -->

- [x] N/A
- [ ] This PR affects the output files or results.
    - [ ] User feedback is considered unnecessary because [Justification].
    - [ ] Affected users have been included in the development process and given a chance to provide feedback.

### Infrastructure Changes

<!-- Link related issues or PRs for necessary changes. -->
<!-- Make sure that the linked PRs include relevant tests. -->

- **Stored files in Housekeeper**
    - [x] N/A
    - [ ] Updated: [Link]
- **CG (CLI and delivered/uploaded files)**
    - [x] N/A
    - [ ] Updated: [Link]
- **Servers (configuration files on Hasta)**
    - [x] N/A
    - [ ] Updated: [Link]
- **Scout interface**
    - [x] N/A
    - [ ] Updated: [Link]

### Validation criteria

<!-- Create validation criteria for this PR or mark as unnecessary. -->

Validation criteria to be added to validation report PR: [LINK-TO-VALIDATION-REPORT-PR from the [validations repository](https://github.com/Clinical-Genomics/validations/)]

**Version specific criteria**

- Text here or N/A



> [!IMPORTANT]
> One of the below checkboxes for validation need to be checked 

- [ ] Added version specific validation criteria to validation report
- [ ] Changes validated in standard sections: [validation-section]
- [ ] Validation criteria not necessary

## Checklist

> [!IMPORTANT]  
> Ensure that all checkboxes below are ticked before merging.

### For Developers

- **PR Description**
    - [ ] Provided a comprehensive description of the PR.
    - [ ] Linked relevant user stories or issues to the PR.
- **Documentation**
    - [ ] Verified and updated documentation if necessary.
- **Validation criteria**
    - [ ] Completed the validation criteria section of the template.
- **Tests**
    - [ ] Described and tested the functionality addressed in the PR.
    - [ ] Ensured integration of the new code with existing workflows.
    - [ ] Confirmed that meaningful unit tests were added for the changes introduced.
    - [ ] Checked that the PR has successfully passed all relevant code smells and coverage checks.
- **Review**
    - [ ] Addressed and resolved all the feedback provided during the code review process.
    - [ ] Obtained final approval from designated reviewers.

### For Reviewers

- **Code**
    - [ ] Code implements the intended features or fixes the reported issue.
    - [ ] Code follows the project's coding standards and style guide.
- **Documentation**
    - [ ] Pipeline changes are well-documented in the CHANGELOG and relevant documentation.
- **Validation criteria**
    - [ ] The author has completed the validation criteria section of the template
- **Tests**
    - [ ] The author provided a description of their manual testing, including consideration of edge cases and boundary
      conditions where applicable, with satisfactory results.
- **Review**
    - [ ] Confirmed that the developer has addressed all the comments during the code review.

<!-- Add any other relevant information or specific checks necessary for your PR. -->

mathiasbio reviewed (commented):
Great job 🌟 Code looks good to me, and great that the containers can build! 

There's just a couple of things left that I would love to know. 

First is if the workflows that use these containers can still finish successfully, it's probably enough to just test one workflow, as long as that workflow tests each way that the containers are used. 

Then it would also be nice to understand a bit more about what the impact could be on the VCFs from updating cadd and vep. 

- Is the format of the VCFs changed in any way?
- Is there new information added to the info fields?
- Has existing fields changed names?
- If anything has changed, can Scout still parse it?
- If there's new info, is there anything that would be relevant to ask Scout devs to add?

It can be difficult to understand all of the changes of course! But for one it would be nice to see if the second breakpoint in the SVs is now annotated for the fusion detection in Scout :  )

fevac commented:
@mathiasbio Addressing your comments:

- Is the format of the VCFs changed in any way?
Same format. With some new information (see below) and updated DBs.

The most relevant updates are:
* Breakend improvements
    * Use of chromosome synonyms supported in breakends
    * Consequences now reported per breakend
    * Support for single breakend input

- Is there new information added to the info fields?
Some new fields, mostly from gnomAD
<img width="1454" alt="Pasted Graphic 1" src="https://github.com/user-attachments/assets/5be4a26d-3372-41e5-a6fc-630663bed78b" />

Has existing fields changed names?
AA_AF and EA_AF have been removed

If anything has changed, can Scout still parse it?
Testing

If there's new info, is there anything that would be relevant to ask Scout devs to add?
I think nothing for now

fevac commented:
Tested scout upload: it works without issues. SNVs and SVs can be seen as usual

rannick reviewed (approved):
LGTM!
